video-intelligence-label-detection/extension.yaml

```diff
@@ -0,0 +1,144 @@
+    validationRegex: ^([\/0-9a-z_.-*]*)$
+    validationErrorMessage: Invalid storage folder name
+    default: ${STORAGE_BUCKET}/video_annotations_input/*
+    required: true
+
+  - param: OUTPUT_STORAGE_URI
+    label: Output storage uri
+    description: >
+      Optional. Location where the output (in JSON format) should be stored
+    type: string
+    example: ${STORAGE_BUCKET}/video_annotations_output/
+    validationRegex: ^([\/0-9a-z_.-]*)$
+    validationErrorMessage: Invalid storage folder name
+    default: ${STORAGE_BUCKET}/video_annotations_output
+    required: true
+
+  - param: LABEL_DETECTION_MODE
+    label: Label detection mode
+    description: >
+      What labels should be detected with LABEL_DETECTION, in addition to video-level labels or segment-level labels. If unspecified, defaults to SHOT_MODE.
+    type: select
+    options:
+      - label: Shot mode
+        value: SHOT_MODE
+      - label: Frame mode
+        value: FRAME_MODE
+      - label: Shot and frame mode
+        value: SHOT_AND_FRAME_MODE
+    default: SHOT_MODE
+    required: true
+
+  - param: VIDEO_CONFIDENCE_THRESHOLD
+    label: Video confidence threshold
+    description: >
+      The confidence threshold we perform filtering on the labels from video-level and shot-level detections. If not set, it is set to 0.3 by default. The valid range for this threshold is [0.1, 0.9]. Any value set outside of this range will be clipped. Note: for best results please follow the default threshold. We will update the default threshold everytime when we release a new model.
+    example: 0.3
+    validationRegex: ^[01]\.[0-9]
+    validationErrorMessage: Must be a value between 0.1 and 0.9
+    default: 0.3
+    required: true
+
+  - param: FRAME_CONFIDENCE_THRESHOLD
+    label: Frame confidence threshold
+    description: >
+      The confidence threshold we perform filtering on the labels from frame-level detection. If not set, it is set to 0.4 by default. The valid range for this threshold is [0.1, 0.9]. Any value set outside of this range will be clipped. Note: for best results please follow the default threshold. We will update the default threshold everytime when we release a new model.
+    example: 0.4
+    validationRegex: ^[01]\.[0-9]
+    validationErrorMessage: Must be a value between 0.1 and 0.9
+    default: 0.4
+    required: true
+
+  - param: MODEL
+    label: Model
+    description: >
+      Model to use for label detection.
+    example: builtin/stable
+    options:
+      - label: builtin/stable
+        value: builtin/stable
+      - label: builtin/latest
+        value: builtin/latest
+    default: builtin/latest
+    required: true
+
+  - param: STATIONARY_CAMERA
+    label: Stationary Camera
+    description: >
+      Whether the video has been shot from a stationary (i.e. non-moving) camera. When set to true, might improve detection accuracy for moving objects. Will default to false if SHOT_AND_FRAME_MODE enabled.
```

Comment:
```suggestion
      Whether the video has been shot from a stationary (i.e. non-moving) camera. When set to true this might improve detection accuracy for moving objects. Will default to false if LABEL_DETECTION_MODE has been set to SHOT_AND_FRAME_MODE.
```